content/vendor/quickstart/_index.md

```diff
@@ -1,6 +0,0 @@
----
```

Comment:
added the redirect to the quickstart guide via hugo `aliases` field instead of this index file.